dunecore/Utilities/services_protodunehd.fcl

```diff
@@ -7,35 +7,45 @@ BEGIN_PROLOG
 ### ProtoDUNE-HD           ###
 ##############################
 
-protodunehd_services: @local::protodune_services 
+protodunehd_services: @local::protodune_services
 protodunehd_services.AuxDetGeometry: @local::protodunehdv6_auxdet_geo
 protodunehd_services.Geometry:  @local::protodunehdv6_geo
 protodunehd_services.DetectorClocksService: @local::protodunehd_detectorclocks
+protodunehd_services.LArPropertiesService.ScintPreScale: 1
```

Comment:
Did you compare the size of the output with ScintPreScale set to 1? I remember @ahimmel being against setting this to 1 by default in the past.

Reply:
Hi Laura, it only affects the size when creating simphotons, but we are always running fast simulation. So it doesn't have sense anymore.